src/controllers/UsersController.php

```diff
@@ -386,6 +386,42 @@ public function deleteUser($request, $db)
         $view->setResponseCode(204);
     }
 
+    /**
+     * Allow users to be set as trusted
+     *
+     * @param $request
+     * @param $db
+     *
+     * @return null
+     *
+     * @throws Exception
+     *
+     */
+    public function setTrusted($request, $db)
+    {
+        if (false == ($request->getUserId())) {
+            throw new Exception("You must be logged in to change a user account", 401);
+        }
+
+        $user_mapper = $this->getUserMapper($db, $request);
+        if (!$user_mapper->isSiteAdmin($request->getUserId())) {
+            throw new Exception("You must be an admin to change a user's trusted state", 401);
```

Comment:
403 is forbidden, 401 is Unauthorized - I've never really seen a consensus opinion one way or the other - as far as I understand it, the RFC implies that if a 401 is sent, then correct authentication may help - which means in both these situations a 401 is correct,

From the RFC:

 > If the request already included Authorization credentials, then the 401 response indicates that authorization has been refused for those credentials.

I don't have an issue with changing them, I just re-used an authentication snippet from further up the controller ;-)